src/configs/YITHPlugins.config.js

```diff
@@ -294,5 +295,30 @@ export const YITHPluginsDefinitions = (props) => ({
         },
       ],
     },
+    {
+      Card: FeatureCard,
+      shouldRender: () => true,
+      name: "",
```

Comment:
You can add name, its not necessary that name should be same as slug

Reply:
added the name